Traceback (most recent call last):
  File "/home/rockettm/catkin_ws/src/mushr_rhc/mushr_rhc_ros/src/main.py", line 9, in <module>
    node.start("rhcontroller")
  File "/home/rockettm/catkin_ws/src/mushr_rhc/mushr_rhc_ros/src/rhcnode.py", line 67, in start
    next_ctrl = self.rhctrl.step(ip)
  File "/home/rockettm/catkin_ws/src/mushr_rhc/mushr_rhc_ros/src/librhc/mpc.py", line 61, in step
    costs = self.cost.apply(self.rollouts, self.goal)
  File "/home/rockettm/catkin_ws/src/mushr_rhc/mushr_rhc_ros/src/librhc/cost/waypoints.py", line 53, in apply
    all_poses).view(self.K, self.T)
  File "/home/rockettm/catkin_ws/src/mushr_rhc/mushr_rhc_ros/src/librhc/worldrep/simple.py", line 101, in check_collision_in_map
    self.perm |= self.perm_reg[bbox_idx[:, 1, 0], bbox_idx[:, 0, 0]]
RuntimeError: invalid argument 2: out of range: 8 out of 25375878 at /opt/conda/conda-bld/pytorch_1524577177097/work/aten/src/TH/generic/THTensorMath.c:430
[rhcontroller-2] process has died [pid 1557, exit code 1, cmd /home/rockettm/catkin_ws/src/mushr_rhc/mushr_rhc_ros/src/main.py __name:=rhcontroller __log:=/home/rockettm/.ros/log/6c4746e8-0492-11e9-9dd0-704d7b6f487c/rhcontroller-2.log].
log file: /home/rockettm/.ros/log/6c4746e8-0492-11e9-9dd0-704d7b6f487c/rhcontroller-2*.log


